primitives/manta/src/lib.rs

```diff
@@ -14,12 +14,17 @@
 // You should have received a copy of the GNU General Public License
 // along with Manta.  If not, see <http://www.gnu.org/licenses/>.
 
-#![allow(non_upper_case_globals)]
-#![allow(clippy::unnecessary_cast)]
-#![allow(clippy::upper_case_acronyms)]
+//! Manta Primitives
+
 #![cfg_attr(not(feature = "std"), no_std)]
+#![cfg_attr(doc_cfg, feature(doc_cfg))]
+#![forbid(rustdoc::broken_intra_doc_links)]
+#![forbid(missing_docs)]
+
+extern crate alloc;
 
 pub mod assets;
 pub mod constants;
 pub mod types;
+///
```

Comment:
why this empty doccomment?

Reply:
forgotten, will add

Reply:
Oh no, just move the comment to the XCM file